### ISSUE TITLE
[Bug]: Broken symlinks "neo4j-shared-templates" at https://neo4j.com/download-center/#helm

### ISSUE THREAD

**Issue by johnlinp** [neo4j/helm-charts#87]
### Contact Details

johnlinp@gmail.com

### What happened?

When I downloaded the release tarball at https://neo4j.com/download-center/#helm and extracted it, I found some broken symlinks in the Helm charts. If I go ahead install the chart, an error will occur:

```
Error: INSTALLATION FAILED: error evaluating symlink /neo4j-standalone/charts/neo4j-shared-templates: lstat /neo4j-standalone/../neo4j-shared-templates: no such file or directory
```

The broken symlinks are:

```
$ ls -l */charts
neo4j-cluster-core/charts:
total 0
lrwxr-xr-x  1 29 Sep  8 03:34 neo4j-shared-templates -> ../../neo4j-shared-templates/

neo4j-cluster-headless-service/charts:
total 0
lrwxr-xr-x  1 29 Sep  8 03:34 neo4j-shared-templates -> ../../neo4j-shared-templates/

neo4j-cluster-loadbalancer/charts:
total 0
lrwxr-xr-x  1 29 Sep  8 03:34 neo4j-shared-templates -> ../../neo4j-shared-templates/

neo4j-cluster-read-replica/charts:
total 0
lrwxr-xr-x  1 28 Sep  8 03:34 neo4j-shared-templates -> ../../neo4j-shared-templates

neo4j-standalone/charts:
total 0
lrwxr-xr-x  1 29 Sep  8 03:34 neo4j-shared-templates -> ../../neo4j-shared-templates/

$ ls neo4j-standalone/charts/neo4j-shared-templates/
ls: neo4j-standalone/charts/neo4j-shared-templates/: No such file or directory
```

### Chart Name

All

### Chart Version

4.4.10

### Environment

Local Machine

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by harshitsinghvi22** [comment]
Hi @johnlinp 

This is fixed now in the 4.4.12 and release onwards.

Thanks,
Harshit

**Comment by johnlinp** [comment]
Thank you!
